src/main/java/com/epam/training/designpatterns/fastfoodrestaurant/config/AppConfig.java

```diff
@@ -0,0 +1,56 @@
+package com.epam.training.designpatterns.fastfoodrestaurant.config;
+
+import com.epam.training.designpatterns.fastfoodrestaurant.entities.Restaurant;
+import com.epam.training.designpatterns.fastfoodrestaurant.simulation.RestaurantSimulator;
+import com.epam.training.designpatterns.fastfoodrestaurant.workstations.Chef;
+import com.epam.training.designpatterns.fastfoodrestaurant.workstations.DeliveryQueue;
+import com.epam.training.designpatterns.fastfoodrestaurant.workstations.OrderQueue;
+import com.epam.training.designpatterns.fastfoodrestaurant.workstations.Waiter;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.*;
+import org.springframework.core.env.Environment;
+
+@Configuration
+@PropertySource("/restaurant.properties")
+public class AppConfig {
+
+    @Autowired
+    Environment environment;
```

Comment:
Lehet `private`. Illetve ha csak property-kre van szükséged, akkor `@Value`-val is be lehet őket injektálni, nem kell az egész `Environment`. Persze jó így is.